sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -197,7 +199,10 @@ def _map_callable_to_attr(self, *, attr: str, func: Callable):
             return
 
         module = inspect.getmodule(func)  # Determine module / function name
-        if module:
+        if in_jupyter():
```

Comment:
can we make the jupyter case the non-default. I.e. move this to the else condition